matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/identity/model/SignInvitationResult.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2021 The Matrix.org Foundation C.I.C.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.matrix.android.sdk.internal.session.identity.model
+
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+data class SignInvitationResult(
```

Comment:
not internal, but can we improve the API to avoid exposing this data to the application?

Reply:
I added some comments, but we should soon remove this API as we should not need it. It's a helper for  clients who may not be able to perform crypto themselves. But the I wasn't able to find details on what crypto need to be done (opened a ticket for that, as well as updated a sydent issue to say that this v1 API can't be removed yet https://github.com/matrix-org/sydent/issues/338#issuecomment-886632038)